### PR TITLE
Update ember

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Aiming to resolve: https://github.com/tastejs/todomvc/issues/2255

How do you run the meta app?
I see engines is set to node 8...?

### CODE REVIEW

**Comment by flashdesignory** [comment]
@NullVoxPopuli - you're right, the site needs node 8 to run with gulp.
I believe it's `npm run gulp serve` from root.

(@FadySamirSadek - in case I am forgetting something here)

**Comment by NullVoxPopuli** [comment]
I can't get it to run:
```bash
❯ npm run gulp serve

> todomvc@0.1.2 gulp
> gulp serve

fs.js:43
} = primordials;
    ^

ReferenceError: primordials is not defined
    at fs.js:43:5
    at req_ (<repo>/node_modules/natives/index.js:137:5)
    at Object.req [as require] (<repo>/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (<repo>/node_modules/gulp/node_modules/graceful-fs/fs.js:1:37)

```

**Comment by flashdesignory** [comment]
@NullVoxPopuli  - are you using node 8.17.0?
You might have to run install again after switching, but the command is pretty much what I'm using.

**Comment by NullVoxPopuli** [comment]
install never finishes

File: learn.json [hunk 1]
```diff
@@ -423,17 +423,41 @@
...
 			"heading": "Community",
 			"links": [{
+				"name": "Discord",
+				"url": "https://discord.gg/emberjs"
+			}, {
+				"name": "BlueSky",
+				"url": "https://bsky.app/profile/emberjs.com"
+			}, {
+				"name": "Mastodon",
+				"url": "https://hachyderm.io/@emberjs",
```
Comment (flashdesignory) [inline learn.json:451]: nit: trailing comma

**Review by flashdesignory — CHANGES_REQUESTED** [review]
in your learn.json file, please update "emberjs" (on line 403) to just "ember". 
In that way it matches your index.html file (<!DOCTYPE html><html lang="en" data-framework="ember">...).

I was able to pull your branch, install and run it. Not sure why it's not working for you locally... the old node version might be a bit wonky to work with 🤷 .

**Comment by NullVoxPopuli** [comment]
thanks, updated!

**Comment by NullVoxPopuli** [comment]
>  Not sure why it's not working for you locally

I couldn't find what combination of npm/node to use -- I suspect that if I have the right combo of versions, I could run things

**Pull request merged**